bundles/binding/org.openhab.binding.zwave/database/rtc/ct32.xml

```diff
@@ -0,0 +1,221 @@
+			<Item>
+				<Value>1</Value>
+				<Label lang="en">0.0F</Label>
+			</Item>
+			<Item>
+				<Value>2</Value>
+				<Label lang="en">1F</Label>
+			</Item>
+			<Item>
+				<Value>3</Value>
+				<Label lang="en">1.5F</Label>
+			</Item>
+						<Item>
+				<Value>4</Value>
+				<Label lang="en">2F</Label>
+			</Item>
+						<Item>
+				<Value>5</Value>
+				<Label lang="en">2.5F</Label>
+			</Item>
+						<Item>
+				<Value>6</Value>
+				<Label lang="en">3F</Label>
+			</Item>
+						<Item>
+				<Value>7</Value>
+				<Label lang="en">3.5F</Label>
+			</Item>
+						<Item>
+				<Value>8</Value>
+				<Label lang="en">4F</Label>
+			</Item>
+		</Parameter>
+		<Parameter>
+			<Index>8</Index>
+			<Type>int</Type>
+			<Minimum>4</Minimum>
+			<Maximum>3084</Maximum>
+			<Default>4</Default>
+			<Size>2</Size>
+			<Label lang="en">Thermostat Diff Temperature</Label>
+            <Help lang="en">The Thermostat Differential Temperature configuration command sets the differential temperature for multi-stage HVAC systems. The differential temperature delta defines when the thermostat will turn on additional stages. There are two differential temperatures, one for multistage cool systems and one for multistage heat systems. If the thermostat is not configured for multistage HVAC systems then these parameters have no effect.</Help>
+		</Parameter>
+		<Parameter>
+			<Index>9</Index>
+			<Type>list</Type>
+			<Default>2</Default>
+			<Size>1</Size>
+			<Label lang="en">Thermostat Recovery Mode</Label>
+            <Help lang="en">This parameter defines the thermostat recovery mode.</Help>
+			<Item>
+				<Value>1</Value>
+				<Label lang="en">Recovery Mode is set to fast.</Label>
+			</Item>
+			<Item>
+				<Value>2</Value>
+				<Label lang="en">Recovery Mode is set to economy.</Label>
+			</Item>
+		</Parameter>
+		<Parameter>
+			<Index>11</Index>
+			<Type>list</Type>
+			<Default>1</Default>
+			<Size>1</Size>
+			<Label lang="en">Simple UI Mode Enable/Disable</Label>
+            <Help lang="en">If set to 0, Simple UI Mode is disabled, if set to 1, will enable the Simple UI Mode.</Help>
+			<Item>
+				<Value>0</Value>
+				<Label lang="en">Function is disabled.</Label>
+			</Item>
+			<Item>
+				<Value>1</Value>
+				<Label lang="en">Function is enabled.</Label>
+			</Item>
+		</Parameter>
+			<Parameter>
+			<Index>12</Index>
+			<Type>list</Type>
+			<Default>0</Default>
+			<Size>1</Size>
+			<Label lang="en">Multicast Enable/Disable</Label>
+            <Help lang="en">If set to 0, multicast is disabled, if set to 1, will enable the multicast.</Help>
+			<Item>
+				<Value>0</Value>
+				<Label lang="en">Function is disabled.</Label>
+			</Item>
+			<Item>
+				<Value>1</Value>
+				<Label lang="en">Function is enabled.</Label>
+			</Item>
+		</Parameter>
+	</Configuration>
+
+	<Associations>
+		<Group>
+			<Index>1</Index>
```

Comment:
Since this group looks like it's to advise the controller about state, please add the SetToController tag to true.